### PR TITLE
fixed warning on node versions more then 0.10

### PR DESCRIPTION
**Pull request by dderg**
also added more node versions to test in travis.yml

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 language: node_js
 node_js:
   - '0.10'
+  - '0.12'
+  - '4.0'
+  - '4.1'
+  - 'stable'
```
Comment (ben-eb) [inline .travis.yml]: `stable` is fine, don't need `4.0` & `4.1` as well.

**Comment by prog666** [comment]
fixed

**Comment by ben-eb** [comment]
Thanks. :+1:

https://github.com/frontendfriends/node-combine-mq/commit/70124efd0911d917cbb5d162464485527a1772db

**Comment by dderg** [comment]
can you "npm publish", please? I still get warnings

**Comment by ben-eb** [comment]
You'll want @furzeface for that. :smile:

**Comment by dderg** [comment]
ok, hope he sees this topic

**Pull request closed without merging**